src/editors/selectize.js

```diff
@@ -1,106 +1,113 @@
-import { SelectEditor } from './select'
-import { $extend } from '../utilities'
-export var SelectizeEditor = SelectEditor.extend({
+import  { SelectEditor } from  './select.js'
```

Comment:
File modes are changed to 755, they should be 644

Reply:
Not sure why but that's an easy fix.